src/command_frame.cc

```diff
@@ -21,51 +34,84 @@ auto intelligence::jsonschema::cli::frame(
                                  resolver(options))
       .wait();
 
-  for (const auto &[key, entry] : frame) {
-    std::cout << "(LOCATION) URI: ";
-    std::cout << key.second << "\n";
-    std::cout << "    Schema           : " << entry.root.value_or("<ANONYMOUS>")
-              << "\n";
-    std::cout << "    Pointer          :";
-    if (!entry.pointer.empty()) {
-      std::cout << " ";
-    }
+  const auto output_json = options.contains("json") || options.contains("j");
+  if (output_json) {
+    auto output_json_object = sourcemeta::jsontoolkit::JSON::make_object();
+    auto frame_json = sourcemeta::jsontoolkit::JSON::make_object();
+    auto references_json = sourcemeta::jsontoolkit::JSON::make_object();
 
-    sourcemeta::jsontoolkit::stringify(entry.pointer, std::cout);
-    std::cout << "\n";
-    std::cout << "    Base URI         : " << entry.base << "\n";
-    std::cout << "    Relative Pointer :";
-    if (!entry.relative_pointer.empty()) {
-      std::cout << " ";
+    for (const auto &[key, entry] : frame) {
+      auto frame_entry = sourcemeta::jsontoolkit::JSON::make_object();
+      frame_entry.assign("schema", sourcemeta::jsontoolkit::JSON{entry.root.value_or("<ANONYMOUS>")});
```

Comment:
If the root is not defined, can you make it null rather than setting it to a string with the value of `<ANONYMOUS>`?